cli/medperf/tests/commands/mlcube/test_list.py

```diff
@@ -0,0 +1,101 @@
+from typing import Any
+
+import pytest
+
+from medperf.entities.cube import Cube
+from medperf.commands.list import EntityList
+
+PATCH_CUBE = "medperf.entities.cube.Cube.{}"
+
+
+def generate_cube(id: int, is_valid: bool, owner: int) -> dict[str, Any]:
+    git_mlcube_url = f"{id}-{is_valid}-{owner}"
+    name = git_mlcube_url
+    return {
+        'id': id,
+        'is_valid': is_valid,
+        'owner': owner,
+        'git_mlcube_url': git_mlcube_url,
+        'name': name
+    }
+
+
+def cls_local_cubes(*args, **kwargs) -> list[Cube]:
+    return [
+        Cube(**generate_cube(id=101, is_valid=True, owner=1)),
+        Cube(**generate_cube(id=102, is_valid=False, owner=1)),
+        # Intended: for local mlcubes owner is never checked.
+        # All local cubes are supposed to be owned by current user
+
+        # generate_cube(id=103, is_valid=True, owner=12345),
+        # generate_cube(id=104, is_valid=False, owner=12345),
+    ]

```

Comment:
The current behavior is not to filter local cubes by owner. I didn't change it, instead I anchored it in this test case. In future, once we elaborate & implement new `ls` behavior, these tests have to be changed